Modules/Numerics/Optimizersv4/src/itkLBFGS2Optimizerv4.cxx

```diff
@@ -274,6 +293,7 @@ LBFGS2Optimizerv4::GetStopConditionDescription() const
 }
 
 // Disallow setting scale and weight
+/*
```

Comment:
Why not remove this now?

Reply:
Yes should be removed.

Reply:
Removed in commit
https://github.com/InsightSoftwareConsortium/ITK/pull/2372/commits/0431e2297516b43f5f83e2100bc11b04dd420a65